modules/apps/static/portal-configuration/portal-configuration-metatype/src/main/java/com/liferay/portal/configuration/metatype/util/ConfigurationScopedPidUtil.java

```diff
@@ -0,0 +1,126 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.portal.configuration.metatype.util;
+
+import com.liferay.portal.configuration.metatype.annotations.ExtendedObjectClassDefinition;
+import com.liferay.portal.kernel.util.StringBundler;
+import com.liferay.portal.kernel.util.StringUtil;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * @author Drew Brokke
+ */
+public class ConfigurationScopedPidUtil {
+
+	public static final String PID_SCOPE_SEPARATOR_COMPANY = "__COMPANY__";
+
+	public static final String PID_SCOPE_SEPARATOR_GROUP = "__GROUP__";
+
+	public static final String PID_SCOPE_SEPARATOR_PORTLET_INSTANCE =
+		"__PORTLET_INSTANCE__";
+
+	public static String buildEncodedPid(
```

Comment:
I prefer calling this `getConfigurationScopedPid`. That name is more clear and follows the pattern with `ConfigurationPidUtil.getConfigurationPid`. What do you think?

Reply:
I agree. 👍